docs/content/direct/wds.md

```diff
@@ -0,0 +1,8 @@
+# Workload Description Spaces
+
+This document will tell users what they need to know about these.
+
+Including the fact that the KubeFlex hosting cluster can play the role
+of WDS, and how to make that happen.
+
+TODO: write this.
```

Comment:
Can we add the under construction image?  Seems like a common thing to do

Reply:
Due to meetings, I can not get this done before your day is over. Could we address that in follow-on?